src/ui/atoms/theme/ThemeSwitcher.tsx

```diff
@@ -27,8 +26,7 @@ export const ThemeSwitcher: React.FC<ThemeSwitcherProps> = ({
   const { theme, setTheme }: ThemeContextType = useTheme()
   const [value, setValue]: LocalStorageState = useLocalStorage(localStorageKey)
   const prefersColorDark: boolean = useMediaType('(prefers-color-scheme: dark)')
-
-  const isDarkTheme = theme === 'dark'
+  const icon = theme === 'light' ? 'moon-dark' : 'sun-dark'
```

Comment:
It's weak regarding the type (if you make a typo in the name, you have no protection). I guess icon should be an `IconName` somehow.

Reply:
I`ve added it to my last commit 😜